demo/src/main/java/com/google/maps/android/utils/demo/CustomMarkerClusteringDemoActivity.java

```diff
@@ -125,8 +125,7 @@ protected void onClusterUpdated(@NonNull Cluster<Person> cluster, Marker marker)
          * @return a BitmapDescriptor representing a cluster
          */
         private BitmapDescriptor getClusterIcon(Cluster<Person> cluster) {
-            List<Drawable> profilePhotos;
-            profilePhotos = new ArrayList<Drawable>(Math.min(4, cluster.getSize()));
+List<Drawable> profilePhotos = new ArrayList<>(Math.min(4, cluster.getSize()));
```

Comment:
Everything Changes I Thinking About LoGo this Because in The Future It's Nam  All Connect. But now Please Please make application for Test with My City  For see about should What to make new Hope You Understand